### ISSUE TITLE
Keycloak: Configure Realm

### ISSUE THREAD

**Issue by Morphclue** [Morphclue/apollusia#159]
# Context
Basic settings for Apollusia realm with Social Identity Providers.

- [x] Google
- [ ] Facebook
- [x] GitHub
- [ ] Twitter / X ?

**Comment by 007jithinjose** [comment]
@Morphclue is anyone doing this ? if not can i get assigned this

**Comment by Morphclue** [comment]
@007jithinjose Yeah, I'm currently assigned and will implement it the next days since I will link it to my Dev Accounts for the social providers.

**Comment by 007jithinjose** [comment]
ok thanks for info

**Comment by Morphclue** [comment]
GitHub added, closing for now. If we ever need Facebook / Twitter, please re-open.
